src/Database/Table/Selection.php

```diff
@@ -227,6 +227,18 @@ public function fetch()
 
 
 	/**
+	 * Fetches single field.
+	 * @param  int
+	 * @return mixed|FALSE
+	 */
+	public function fetchField($column = 0)
+	{
+		$row = $this->fetch();
```

Comment:
Makes sense, but shouldn't be then select added to `fetchPairs` too? And it will probably not work for int column key which is default for `fetchField` function.